### ISSUE TITLE
[QUERY] NGINX

### ISSUE THREAD

**Issue by AkzXrated** [navilg/media-stack#58]
Hello I just set up the media stack with VPN thanks to this script.
Kindly help me setting up NGINX.

Right now everything is running just fine and VPN is running too. It took me some efforts as a beginner to get this far.
So I Don't want to break anything that is already working.

But discovered that with the use of NGINX I don't need to input address like 
```
http://192.168.1.200:5080
http://192.168.1.200:7878 etc.
```
How do I deploy NGINX at this point ?
running 
`docker compose -f docker-compose-nginx.yml up`
and following the instructions from 
**Configure Nginx** and what's written below will work without breaking anything ?
Thank you and sorry for this beginner question

**Comment by navilg** [comment]
Yes it will work. Just be aware for the notes in document for VPN enabled env.

Also, Make sure to keep a backup of nginx config file, so that even if it breaks, You can replace the changes with backup file inside volume and restart the nginx container
